runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigureTask.java

```diff
@@ -220,12 +201,27 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
 
             namespace.id = supplyId.applyAsInt(namespace.name);
 
+            namespace.readURL = configURL ->
+            {
+                try
+                {
+                    return this.readURL(configURL);
+                }
+                catch (IOException | URISyntaxException | InterruptedException e)
+                {
+                    rethrowUnchecked(e);
```

Comment:
Please move `rethrowUnchecked(...)` behavior inside `readURL` and use `ex` for exception instead of `e` (by convention).